docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+MAINTAINER koalaman <koalaman@gmail.com>
+
+RUN apt-get update -qy
```

Comment:
apt-get update and apt-get install should be combined in the same run statement.  See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get